src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3216,30 +3263,19 @@ bool SvgLoader::resize(Paint* paint, float w, float h)
     auto sx = w / this->w;
     auto sy = h / this->h;
 
-    if (preserveAspect) {
-        //Scale
-        auto scale = sx < sy ? sx : sy;
-        paint->scale(scale);
-        //Align
-        auto tx = 0.0f;
-        auto ty = 0.0f;
-        auto tw = this->w * scale;
-        auto th = this->h * scale;
-        if (tw > th) ty -= (h - th) * 0.5f;
-        else tx -= (w - tw) * 0.5f;
-        paint->translate(-tx, -ty);
-    } else {
-        //Align
-        auto tx = 0.0f;
-        auto ty = 0.0f;
-        auto tw = this->w * sx;
-        auto th = this->h * sy;
-        if (tw > th) ty -= (h - th) * 0.5f;
```

Comment:
@JSUYA I can't find any explanation why we had this condition.. do u have some idea?

Reply:
It is expected that the code added for center-aligned scale in 20de2bfc153252531b8186972f2d1a36de240474

Reply:
I see, but it was an old logic, when we kinda mixed in this function the general concept of resizing and the transformations between the viewPort and the viewBox.
 
Now, I think, the logic should be that the preserveAspectRatio is take into account only while the scene is built. After that, we have a picture with some width and height. and if we want to squeeze this picture, we should be able to do it, regardless of the svg's aspect ratio. if we want to keep the proportion, we can use scale instead of resize.  

@hermet could you please comment on that? 
I'd keep only these two lines from this function:
```
        Matrix m = {sx, 0, 0, 0, sy, 0, 0, 0, 1};
        paint->transform(m);
```

Reply:
`xMidYMid` is set to default, so we can change it according to your plan.

Reply:
@mgrudzinska I think we can fix it if it's necessary but please double-check our examples whether the result is different after this.

Reply:
thx for your comments, the #1258 shows, that the resizing shouldn't have anything to do with the aspect ratio. I've changed it